src/block-components/image/index.js

```diff
@@ -161,6 +168,10 @@ Image.Content = props => {
 		hasGradientOverlay={ hasHoverOverlay }
 		hasLightbox={ attributes.imageHasLightbox }
 
+		figcaption={ attributes.figcaptionText }
+		figcaptionShow={ attributes.figcaptionShow }
+		figcaptionClasses={ props.figcaptionClasses }
```

Comment:
I think you mean `figcaptionClassnames`
```suggestion
		figcaptionClassnames={ props.figcaptionClasses }
```